src/placement/pl_map_common.c

```diff
@@ -491,7 +491,7 @@ pl_map_extend(struct pl_obj_layout *layout, d_list_t *extended_list)
 		if (f_shard->fs_status != PO_COMP_ST_DRAIN)
 			new_shards[grp_idx].po_rebuilding = 1;
 
-		if (f_shard->fs_status == PO_COMP_ST_UP)
+		if (f_shard->fs_status == PO_COMP_ST_UP || f_shard->fs_status == PO_COMP_ST_NEW)
```

Comment:
just confirm, a reintegrating tgt, it should be in PO_COMP_ST_UP state? (see update_one_tgt, case POOL_REINT)
Is it possible to get the NEW shard here?

Reply:
oh, it could be NEW for extending targets.